clients/clients-integration-tests/src/test/java/org/apache/kafka/clients/ClientsTestUtils.java

```diff
@@ -84,6 +85,15 @@ public static void consumeAndVerifyRecords(
         );
     }
 
+    public static void pollUntilTrue(Consumer<byte[], byte[]> consumer,
+                                     Supplier<Boolean> testCondition,
+                                     long waitTimeMs, String msg) throws InterruptedException {
+        TestUtils.waitForCondition(() -> {
+            consumer.poll(Duration.ofMillis(100));
+            return testCondition.get();
+        }, waitTimeMs, msg);
+    }
```

Comment:
Please follow the code style used in this file.
```suggestion
    public static void pollUntilTrue(
        Consumer<byte[], byte[]> consumer,
        Supplier<Boolean> testCondition,
        long waitTimeMs, 
        String msg
    ) throws InterruptedException {
        TestUtils.waitForCondition(() -> {
            consumer.poll(Duration.ofMillis(100));
            return testCondition.get();
        }, waitTimeMs, msg);
    }
```